### ISSUE TITLE
Events

### ISSUE THREAD

**Issue by PLUCS2** [beccaburten/eventech#5]
- [ ] On heroku there is not a modal to cancel a ticket - just an alert - but when you refresh the ticket page, the ticket is still there 

Excellent work!

**Comment by beccaburten** [comment]
Yes! This was deployed prior to our meeting yesterday to decide on the modal, so it should be fixed on next deployment.
